### PR TITLE
Tf2

### PR DESCRIPTION
**Pull request by moskey71**

### CODE REVIEW

File: modules/rdgw/outputs.tf [hunk 1]
```diff
@@ -1,19 +1,36 @@
 output "rdgw-load-balancer-name" {
-  value = "${lookup(data.aws_cloudformation_stack.this.outputs, "LoadBalancerName", "")}"
+  value = lookup(
+    data.aws_cloudformation_stack.this.outputs,
+    "LoadBalancerName",
+    "",
```
Comment (confusdcodr) [inline modules/rdgw/outputs.tf]: This can just return `null`

File: modules/rdgw/outputs.tf [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+  )
 }
 
 output "rdgw-load-balancer-dns" {
-  value = "${lookup(data.aws_cloudformation_stack.this.outputs, "LoadBalancerDns", "")}"
+  value = lookup(
+    data.aws_cloudformation_stack.this.outputs,
+    "LoadBalancerDns",
+    "",
```
Comment (confusdcodr) [inline modules/rdgw/outputs.tf]: This can just return `null` by default

File: modules/rdgw/outputs.tf [hunk 3]
```diff
@@ -1,19 +1,36 @@
...
-  value = "${aws_route53_record.this.zone_id}"
+  value = aws_route53_record.this.zone_id
 }
 
 output "rdgw-alb-security-group-id" {
-  value = "${lookup(data.aws_cloudformation_stack.this.outputs, "AlbSecurityGroupId", "")}"
+  value = lookup(
+    data.aws_cloudformation_stack.this.outputs,
+    "AlbSecurityGroupId",
+    "",
```
Comment (confusdcodr) [inline modules/rdgw/outputs.tf]: This can just return `null` by default

File: modules/rdgw/outputs.tf [hunk 4]
```diff
@@ -1,19 +1,36 @@
...
+  )
 }
 
 output "rdgw-ec2-security-group-id" {
-  value = "${lookup(data.aws_cloudformation_stack.this.outputs, "Ec2SecurityGroupId", "")}"
+  value = lookup(
+    data.aws_cloudformation_stack.this.outputs,
+    "Ec2SecurityGroupId",
+    "",
```
Comment (confusdcodr) [inline modules/rdgw/outputs.tf]: This can just return `null` by default

File: modules/guac/main.tf [hunk 5]
```diff
@@ -1,46 +1,48 @@
...
+    "SslCertificateService"    = var.SslCertificateService
+    "URL1"                     = var.URL1
+    "URL2"                     = var.URL2
+    "URLText1"                 = var.URLText1
+    "URLText2"                 = var.URLText2
+    "UpdateSchedule"           = var.UpdateSchedule
+    "VPC"                      = var.Vpc
   }
 
   timeouts {
```
Comment (userhas404d) [inline modules/guac/main.tf]: might be better to define these inputs as separate vars. if the desire is to set all inputs with the same value you could have a conditional that if a var is set (something like `CFNGloablTimeoutInMinutes`) then set all inputs to that value, else, use the individual values. Alternatively, recommend moving the `"${var.TimeoutInMinutes}m"` string to a local

File: modules/rdcb/main.tf [hunk 6]
```diff
@@ -51,7 +51,7 @@ resource "aws_security_group" "rdcb-sg1" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
-  tags {
+  tags = {
```
Comment (userhas404d) [inline modules/rdcb/main.tf]:
```suggestion
  tags = {
    merge(map("Name", "${var.StackName}-rdcb-sg1"), var.tags)
  }
```

File: modules/guac/main.tf (hunk 5 above)
Comment (moskey71) [reply modules/guac/main.tf]: Good point.  I'll change the timeouts use separate variables

**Comment by moskey71** [comment]
> Need to add some test cases to this. You can find a starter test case in the `tests/` folder. There is a golang script that will run through each folder in the `tests/` directory and build up the infrastructure and then immediately tear it back down.



> Need to add some test cases to this. You can find a starter test case in the `tests/` folder. There is a golang script that will run through each folder in the `tests/` directory and build up the infrastructure and then immediately tear it back down.

Thanks for the feedback.  I'll address this in a separate PR

File: modules/rdcb/main.tf (hunk 6 above)
Comment (moskey71) [reply modules/rdcb/main.tf]: Going to break this out into a separate task, and address all of the resources

**Pull request merged**